app/src/androidTest/java/org/mozilla/focus/screengrab/AaHomeScreenshot.java

```diff
@@ -0,0 +1,150 @@
+import android.support.test.espresso.Espresso;
+import android.support.test.espresso.contrib.RecyclerViewActions;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+import android.support.test.uiautomator.By;
+import android.support.test.uiautomator.UiDevice;
+import android.support.test.uiautomator.Until;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mozilla.focus.R;
+import org.mozilla.focus.activity.MainActivity;
+import org.mozilla.focus.annotation.ScreengrabOnly;
+import org.mozilla.focus.helper.BeforeTestTask;
+import org.mozilla.focus.utils.AndroidTestUtils;
+import org.mozilla.focus.utils.DialogUtils;
+
+import tools.fastlane.screengrab.Screengrab;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.longClick;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+@ScreengrabOnly
+@RunWith(AndroidJUnit4.class)
+/**
+ * Add 'Aa' prefix in class name to make sure this test is the first one to run
+ **/
+public class AaHomeScreenshot extends BaseScreenshot {
+
+    UiDevice device = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation());
+
+    @Rule
+    public final ActivityTestRule<MainActivity> activityTestRule = new ActivityTestRule<>(MainActivity.class, true, false);
+
+    @Before
+    public void setUp() {
+        new BeforeTestTask.Builder().build().execute();
+        activityTestRule.launchActivity(new Intent());
+
+    }
+
+    @Test
+    public void screenshotHomeFragment() {
+
+        // Remove top site button
+        onView(withId(R.id.main_list)).perform(RecyclerViewActions.actionOnItemAtPosition(0, longClick()));
+        Screengrab.screenshot(ScreenshotNamingUtils.HOME_REMOVE_TOP_SITE);
+        Espresso.pressBack();
+
+        // Home menu
+        AndroidTestUtils.tapHomeMenuButton();
+        Screengrab.screenshot(ScreenshotNamingUtils.HOME_MENU);
+
+        // Click bookmarks
+        onView(withId(R.id.menu_bookmark)).perform(click());
+        onView(withText(R.string.bookmarks_empty_view_msg)).check(matches(isDisplayed()));
+        Screengrab.screenshot(ScreenshotNamingUtils.HOME_MENU_NO_BOOKMARKS);
+        SystemClock.sleep(MockUIUtils.TAKE_SCREENSHOT_DELAY);
+
+        // Click downloads
+        onView(withId(R.id.downloads)).perform(click());
+        onView(withText(R.string.no_downloads)).check(matches(isDisplayed()));
+        Screengrab.screenshot(ScreenshotNamingUtils.HOME_MENU_NO_DOWNLOAD);
+        SystemClock.sleep(MockUIUtils.TAKE_SCREENSHOT_DELAY);
```

Comment:
I don't like the delay here. Should be a better way for the delay.
Talked to Joesph, This is because Screengrab won't wait for Idling Resources.
Maybe we should use https://developer.android.com/reference/android/support/test/runner/screenshot/Screenshot 
And switch locales using the pref in our settings.